tox.ini

```diff
@@ -1,9 +1,38 @@
 [tox]
-envlist = py34,py35,py36,py37
+envlist =
+    clean
+    django22-py{37,36}
+    django21-py{37,36,35}
+    django20-py{36,35,34}
+    report
```

Comment:
Let's remove clean and report from the envlist. We don't want to run coverage for every environment.